docs/eog/case-studies-cw20-timeline.rst

```diff
@@ -0,0 +1,145 @@
+.. _CW20-Timeline: 
+
+CW20 Timeline
+=================
+
+We made the decision three weeks before CW20 to move the event online. 
+We had less than one month to contact all of our stakeholders (venue, accommodation, sponsors, speakers, workshop facilitators and of course participants), inform them of the decision and renegotiate everything that had been previously agreed. 
+As days passed and the implications around the global pandemic became more clear, stakeholders were very understanding and accommodating. 
+Below is a breakdown of what we did. 
+
+Event Project Stage
+--------------------
+
+Month -1 (March 2020)
+^^^^^^^^^^^^^^^^^^^^^^
+
+Agenda
+  - We reorganised agenda for online format by
+   
+    - Removing in-person social programme (walk in Belfast Botanical Gardens, visit to Ulster Museum, Drinks Reception, Conference Dinner).
```

Comment:
This is the first time you mention that the event was originally meant to take place in Belfast (and perhaps it was worth mentioning that this would be the first CW taking place in Northern Ireland) - in the intro of the case study.